pom.xml

```diff
@@ -6,7 +6,7 @@
     <parent>
         <groupId>com.lithium.dbi.rdbi</groupId>
         <artifactId>rdbi-parent</artifactId>
-        <version>0.69</version>
+        <version>0.70-SNAPSHOT</version>
```

Comment:
lol no, when you release this, there's a way to do this

Reply:
i think i messed up something with git